internal/connect/command_logs.go

```diff
@@ -178,7 +178,7 @@ func (c *logsCommand) queryLogs(cmd *cobra.Command, args []string) error {
 		connectorName,
 	)
 
-	logs, err := c.V2Client.SearchConnectorLogs(crn, startTime, endTime, levels, searchText, lastQueryPageToken)
+	logs, err := c.V2Client.SearchConnectorLogs(crn, connectorId, startTime, endTime, levels, searchText, lastQueryPageToken)
```

Comment:
This is marked as a new feature, but it feels like it could be a breaking change.

Does adding the connector ID to the logging search params change the behavior of this command?